spec/workers/gather_o_embed_data_spec.rb

```diff
@@ -16,7 +16,7 @@
       "url" => "https://farm4.static.flickr.com/3123/2341623661_7c99f48bbf_m.jpg"
     }
 
-    @flickr_oembed_url = 'http://www.flickr.com/services/oembed/'
+    @flickr_oembed_url = 'https://www.flickr.com/services/oembed/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.